agent/consul/state/catalog.go

```diff
@@ -1130,13 +1141,20 @@ func (s *Store) AssignManualServiceVIPs(idx uint64, psn structs.PeeredServiceNam
 				filteredIPs = append(filteredIPs, existingIP)
 			}
 		}
+		sort.Strings(filteredIPs)
```

Comment:
Previously we were storing VIPs in whatever order they happened to be in. It seemed silly to not be sorting them.